poetry.lock

```diff
@@ -1620,18 +1620,18 @@ testing = ["flaky (>=3.4.0)", "freezegun (>=0.3.11)", "pathlib2 (>=2.3.3)", "psu
 
 [[package]]
 name = "tox-docker"
-version = "4.1.0"
+version = "4.0.0"
 description = "Launch a docker instance around test runs"
 category = "dev"
 optional = false
 python-versions = "*"
 files = [
-    {file = "tox-docker-4.1.0.tar.gz", hash = "sha256:0317e692dc80f2197eaf9c905dcb8d1d1f9d5bf2686ecfd83c22a1da9d23fb24"},
-    {file = "tox_docker-4.1.0-py2.py3-none-any.whl", hash = "sha256:444c72192a2443d2b4db5766545d4413ea683cc488523d770e2e216f15fa3086"},
+    {file = "tox-docker-4.0.0.tar.gz", hash = "sha256:c0b259a6f14fcb68f8e767926aa1d9e6649eb11862444ba704b5cacf93e4e937"},
+    {file = "tox_docker-4.0.0-py2.py3-none-any.whl", hash = "sha256:84a260b8e8096d2bb5e2115d8a75cac33b183e1b5d0ef341302e64971a562731"},
 ]
 
 [package.dependencies]
-docker = ">=4.0,<7.0"
+docker = ">=2.3.0,<6.0"
```

Comment:
Is there no smaller range than this that will work? That's quite a lot of versions...